test_files/values.alp

```diff
@@ -15,6 +18,20 @@ let test_atom = :nope
 let test_bool = true
 let test_chars = c"x"
 let test_tuple = (100, "100")
+let test_unit = ()
+
+let wrapped_int = AnInt 41
```

Comment:
Quite right, apologies!  I would blame insufficient coffee this morning but really I should have just looked closer :)

@danabr if you want this merged prior to discussion/changes between you and @lepoetemaudit  or prefer I hold off just let me know.  I'm fine either way.